woocommerce/woocommerce-android
# Issue/4806 convert fetch shipment tracking into suspendable

**We'll need to update fluxc hash before we merge this PR.**




Parent issue https://github.com/woocommerce/woocommerce-android/issues/4806

"Do not merge" status - fluxc PR should be merged first and the hash updated

### Description


"FetchShipmentTrackings" action was refactored into a suspendable method - more info on FluxC PR https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2123

This PR is the third of 7/8 PRs - their goal is to transform OrderDetailRepository into a stateless singleton. The only state the repository holds are continuations which won't be necessary as soon as all the methods are using suspendable functions instead of event bus.

### Testing instructions

- Make sure you have the shipment tracking plugin installed
1. Open detail of an order
2. Make sure the shipment tracking section is displayed

### Images/gif

<img src="https://user-images.githubusercontent.com/2261188/133796548-620d88d3-7b1c-4b7c-9ddd-7dc380fff3fc.png" width="250px" />
